docs/design/disqualification.md

```diff
@@ -0,0 +1,77 @@
+
+Further, the node will be demonetized.
+
+> Provided the storage node hasn’t been disqualified, the storage node will be paid by the Satellite for the data it has stored over the course ofthe month, per the Satellite’s records.
+
+A disqualified SNO should quickly stop particiapting with a satellite it is disqualified and demonetized on.  It may remain in Kademlia and potentially found during Overlay Cache Discover, as the kademlia network supports multiple satellites.
+
+One option that may be allowed for disqualified storage nodes is a Graceful Exit.  [Storage Node Payment and Incentives for V3](https://docs.google.com/document/d/1-Pxzk-ad-0QtF6nnTwfgzk8e_-XbBNSDxuRvnbd0QL8/edit#heading=h.rz1ehm5mbeuz) describes this feature:
+
+> When a node operator wants to leave the network, if they just shut down the node, the unavailability of the data on the network can contribute to repair costs.  If instead, the node triggers a function to call a Satellite, request new storage nodes to store the pieces stored on the node being shut down, then directly upload those pieces to the new nodes, file repair would be avoided.
+
+## Design
+
+### Use of Disqualified Nodes
+
+Disqualified nodes may be used during download of typical downloads from uplinks or via repair.  Therefore, their IP must be tracked in the overlay cache.
+
+Disqualified nodes may not be used for upload, therefore they should not be returned from node selection proceses of any sort.  There is no reason to update the statistics of disqualified nodes.
+
+The list of disqualified nodes should change infrequently, but could grow large over time.
+
+### Handling Disqualified Nodes
+
+Disqualification can be handled in our existing SQL implementation by adding a `disqualified` column to the `nodes` table:
+
+```sql
+CREATE TABLE nodes (
+  id bytea NOT NULL,
+  ...
+  disqualified boolean NOT NULL,
+  PRIMARY KEY ( id )
+);
+```
+
+Existing SQL queries employing logic such as `WHERE audit_success_ratio >= $2 AND uptime_ratio >= $3` would change to `WHERE disqualified = FALSE`.
+
+Existing calls to the DBX `UpdateNodeInfo()` method must set `disqualified` if appropriate.  This may be best accomplished by a database trigger or stored procedure.
+
+## Rationale
+
+Although disqualification is largely an atomic operation that would be handled well by an external hash, the inherent tie-ins with node selection make the above solution the most straightforward.  If we were to refactor node selection in the future, I would likely leave disqualified nodes out of the stats database, leaving them only in the overlay cache.
+
+## Implementation (Stories)
+
+- Update nodes table using DBX
+- Create nodes table migration scripts
```

Comment:
"Update nodes table using DBX" and "Create nodes table migration scripts" seem like they could be the same story